setup.py

```diff
@@ -11,6 +11,15 @@
 
     'numpy ~= 1.19.4',
     'pandas ~= 1.1.4',
+
+    # for cuda, e.g.: mxnet_cu101 < 2.0.0
+    'mxnet < 2.0.0',
```

Comment:
We'll probably want to make it easy for people to install nprintML with their preferred CUDA-friendly distro of mxnet, (or at least document this). But that can be dealt with soon enough.